src/lwc/gePaymentGatewayManagement/gePaymentGatewayManagement.html

```diff
@@ -0,0 +1,82 @@
+  ~     COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+  ~     INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+  ~     BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+  ~     LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+  ~     CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+  ~     LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+  ~     ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+  ~     POSSIBILITY OF SUCH DAMAGE.
+  -->
+
+<template>
+    <template if:true={isSuccess}>
+        <c-util-alert-banner theme="success"
+                             icon="utility:success"
+                             size="small"
+                             message="Gateway updated.">
+        </c-util-alert-banner>
+    </template>
+    <template if:true={isError}>
+        <c-util-alert-banner theme="failure"
+                             icon="utility:error"
+                             size="small"
+                             message={errorMessage}>
+        </c-util-alert-banner>
+    </template>
+    <lightning-layout multiple-rows="true">
+        <div if:true={showSpinner}
+             class='spinner-container'>
+            <lightning-spinner alternative-text={CUSTOM_LABELS.messageLoading}
+                               size='large'>
+            </lightning-spinner>
+            <p class='loading-text slds-text-heading_medium'>{loadingText}</p>
+        </div>
+        <lightning-layout-item size="12">
+            <h1 class="slds-text-heading_medium slds-p-top-medium">Elevate Gateway Management</h1>
+        </lightning-layout-item>
+        <lightning-layout-item size="12">
+            <p class="slds-text-body_regular slds-p-vertical_medium">NPSP uses the gateway that was added to Elevate most
+                recently. To override this, click Edit and enter the gateway ID for the one you'd like to use instead.
+                Find the gateway ID in the Elevate Admin Console | Gateway Management.</p>
+        </lightning-layout-item>
+        <template if:true={isReadState}>
+            <lightning-layout-item size="12" class="slds-text-align_center">
+                <lightning-button label="Edit" data-id="editGatewayButton" onclick={handleEdit}></lightning-button>
+            </lightning-layout-item>
+        </template>
+        <lightning-layout-item size="4">
+            <template if:true={isReadState}>
+                <lightning-input variant="label-inline" read-only="true" label="Gateway ID" value={gatewayId}
+                                 type="text"></lightning-input>
+            </template>
+            <template if:true={isEditState}>
+                <lightning-input variant="label-inline" label="Gateway ID" value={gatewayId}
+                                 type="text" data-id="gatewayIdEditField"></lightning-input>
```

Comment:
Is it possible to use only one `lightning-input` in the template here and control the read-only property using the `isReadState` field? (and maybe rename the field to "isReadOnly")

Reply:
@kenlewis-sfdo good idea. Just pushed an update for this!